Algorithm-Caesars-Cipher.md

```diff
@@ -1,6 +1,8 @@
+# Algorithm Caesars Cipher
+
 :triangular_flag_on_post: Remember to use [**`Read-Search-Ask`**](How-to-get-help-when-you-get-stuck) if you get stuck. Try to pair program :busts_in_silhouette: and write your own code :memo:
 
-# :checkered_flag: Problem Explanation:
+## :checkered_flag: Problem Explanation:
```

Comment:
The Explanation should appear in chat at the first call. With `h2`, it won't appear.